package.json

```diff
@@ -7,6 +7,6 @@
         "eslint-config-prettier": "^9.1.0",
         "prettier": "^3.3.3",
         "typescript": "^5.6.2",
-        "typescript-eslint": "^8.5.0"
+        "typescript-eslint": "^8.13"
```

Comment:
Please move unrelated changes to another PR(s)

Reply:
Will do